### PR TITLE
Update to Ember 2.8 family + fix Glimmer2

### PR DESCRIPTION
**Pull request by cibernox**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -21,17 +21,17 @@ matrix:
     - env: EMBER_TRY_SCENARIO=ember-canary
 
 before_install:
-  - export PATH=/usr/local/phantomjs-2.0.0/bin:$PATH
-  - "npm config set spin false"
-  - "npm install -g npm@^3"
```
Comment (cibernox) [inline .travis.yml]: Using node 6 already includes npm3 :)

**Comment by mschinis** [comment]
This one is quite important for adding glimmer2.0 support. Can we get this one merged?

**Comment by cibernox** [comment]
@GavinJoyce any to comment on this?

**Comment by GavinJoyce** [comment]
@cibernox Thanks for the PR, I missed it in my notifications. 

👀

**Comment by GavinJoyce** [comment]
Thanks! 🍻

**Comment by GavinJoyce** [comment]
https://github.com/intercom/ember-href-to/releases/tag/v1.6.0

**Pull request merged**